appengine/endpoints/README.md

```diff
@@ -40,7 +42,7 @@ Open the `swagger.yaml` file and in the `host` property, replace
 
 Then, deploy the sample using `gcloud`:
 
-    gcloud app deploy app.yaml
+    gcloud beta app deploy
```

Comment:
For Endpoints, we need to use beta or the Endpoints related stuff doesn't happen.